lib/stats/stats-counter.h

```diff
@@ -72,6 +73,15 @@ stats_counter_get(StatsCounterItem *counter)
   return result;
 }
 
+static inline gchar *
+stats_counter_get_name(StatsCounterItem *counter)
+{
+  if (counter && counter->name)
+    return counter->name;
```

Comment:
I think checking whether `counter->name` is not `NULL` is superfluous.
When counter is `NULL`, that's an error, IHMO and it's not the same case when a counter does not have a name.